python/cuml/dask/common/comms.py

```diff
@@ -178,6 +175,26 @@ def _func_init_nccl(sessionId, uniqueId):
         print("An error occurred initializing NCCL!")
 
 
+class ListenerThread(threading.Thread):
+    def __init__(self, verbose):
+        threading.Thread.__init__(self)
+        self.listener = ucp.create_listener(_connection_func)
+        self.port = self.listener.port
```

Comment:
`port` seems to be unused?

Reply:
This does actually get used so that workers can query the port of the underlying listener (since ucx-py is setting the port to an unused port).